sdk/iot/hub/inc/az_iot_hub_client.h

```diff
@@ -409,30 +426,36 @@ AZ_NODISCARD az_result az_iot_hub_client_methods_response_publish_topic_get(
  * @brief Gets the MQTT topic filter to subscribe to twin operation responses.
  *
  * @param[in] client The #az_iot_hub_client to use for this call.
- * @param[in] mqtt_topic_filter An empty #az_span with sufficient capacity to hold the MQTT topic
- *                              filter.
- * @param[out] out_mqtt_topic_filter The output #az_span containing the MQTT topic filter.
+ * @param[out] mqtt_topic_filter A char buffer with sufficient capacity to hold the MQTT topic
+ *                              filter. On success, will be `NULL` terminated.
+ * @param[in] mqtt_topic_filter_size The size of the passed buffer. Must be greater than 0.
+ * @param[out] out_mqtt_topic_filter_length The optional output length of the mqtt topic filter. Can
+ * be `NULL`.
  * @return #az_result
  */
-AZ_NODISCARD az_result az_iot_hub_client_twin_response_subscribe_topic_filter_get(
+AZ_NODISCARD az_result az_iot_hub_client_twin_get_response_subscribe_topic_filter(

```

Comment:
I think twin has several functionalities: twin_get, twin_patch, twin_response.
I think the API name should consider the functionality such that all API names end in `get_subscribe_topic_filter`, etc.

Reply:
all have been updated